brands/shop/clothes.json

```diff
@@ -2723,6 +2723,18 @@
       "shop": "clothes"
     }
   },
+  "shop/clothes|Soma": {
+    "countryCodes": ["us"],
+    "tags": {
+      "alt_name": "Soma Intimates",
+      "brand": "Soma",
+      "brand:wikidata": "Q69882213",
+      "brand:wikipedia": "en:Soma",
```

Comment:
This doesn't seem to go to a specific page. If there isn't a wikipedia page you can just leave it out

Reply:
I was a little confused about this—it doesn't link to a Wikipedia page (looks like it's been removed: https://en.wikipedia.org/wiki/BD's_Mongolian_Grill), but I did add a wikidata entry with a logo: https://www.wikidata.org/wiki/Q69890408. The Wikidata page is all we'd need to pull the logo into the nsi, right?

Reply:
The wikidata page is the important part. Just nuke both of those wikipedia pages and everything will be good for the index.